apps/src/templates/progress/progressTypes.js

```diff
@@ -50,13 +50,23 @@ export const levelType = PropTypes.shape({
  * A boolean indicating if a student was paired on a level.
```

Comment:
In the `levelType` definition above, can you add in some of the other props that are sometimes in levels?
driver, navigator, contained, submitLevel, submitted, assessment, passed, user_id

Reply:
sure. similar to your question about level.submitted above, i'll have to audit our use of all these properties though. i think we mostly rely on `progress.submitted` and `level.levelKind === LevelKind.assessment` instead of `level.submitted` and `level.assessment` for example.

Reply:
Oh interesting. Would it make sense to deprecate the others then? i.e. `level.submitted` -> `level.__deprecatedSubmitted`
Maybe as a follow-up task.